publication-doi-commons/src/test/java/no/unit/nva/publication/doi/dynamodb/dao/DynamodbStreamRecordDaoTest.java

```diff
@@ -0,0 +1,163 @@
+package no.unit.nva.publication.doi.dynamodb.dao;
+
+import static nva.commons.utils.JsonUtils.objectMapper;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.javafaker.Faker;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Random;
+import java.util.UUID;
+import no.unit.nva.publication.doi.dto.PublicationStreamRecordTestDataGenerator;
+import no.unit.nva.publication.doi.dto.PublicationType;
+import no.unit.nva.publication.doi.dynamodb.dao.DynamodbStreamRecordDao.Builder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DynamodbStreamRecordDaoTest {
```

Comment:
Tests should be on top of the private methods. In general, a class should be constructed like an article. In the beginning we have the main idea (main functionality) and at the bottom we have the (implementation) details.

Reply:
Ive loaded the NVA code style and it autoformats and move private methods on top in tests. Has this been fixed recently in nva-codestyle that I need to reimport it?